src/V3Stats.cpp

```diff
@@ -74,12 +91,36 @@ class StatsVisitor final : public VNVisitor {
         }
     }
 
+    void addRAMStats(AstNodeModule* nodep) {
+        for (const auto& nodep : m_potentialRams) {
+            AstVar* varp = nodep->varp();
+            AstVarScope* varScopep = nodep->varScopep();
+            int depth = varp->dtypep()->arrayUnpackedElements();
+            int bits = depth * varp->width();
+            if (varScopep->user1() <= 2) {  // Single or double assignment
+                std::ostringstream os;
+                os << "Potential RAM bits, " << varScopep->prettyName() << " (" << depth << "x"
+                   << varp->width() << ")";
```

Comment:
Presumably for debug? Use UINFO(9, ...);

If not for debug (a report) then dump to a new output file, not stdout.

Reply:
Not stdout, this is building the string which is written to PREFIX_stats.txt